tests/test_docker_builder.py

```diff
@@ -0,0 +1,62 @@
+# Copyright The Lightning AI team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+
+import litserve as ls
+from litserve import docker_builder
+
+
+def test_color():
+    assert docker_builder.color("hi", docker_builder.RED) == f"{docker_builder.RED}hi{docker_builder.RESET}"
+
+    expected = f"{docker_builder.INFO} {docker_builder.RED}hi{docker_builder.RESET}"
+    assert docker_builder.color("hi", docker_builder.RED, docker_builder.INFO) == expected
+
+
+EXPECTED_CONENT = f"""FROM python:3.10-slim
```

Comment:
Note, this works with CPU things but I couldn't get it to work with GPU - probably need a variant based on nvidia docker images or similar if we want GPU support

Reply:
Right, I am going to do this in a follow up PR.